openlibrary/templates/merge_queue/merge_queue.html

```diff
@@ -38,13 +39,75 @@ <h1>$_('Community Edit Requests')</h1>
   $:macros.Pager(page, total[mode], results_per_page=25)
 </div>
 
-  <ul class="nav-bar">
-    <li class="$(mode=='open' and 'selected' or '')">
-      <a href="$changequery(mode='open', page=None)">$_('Open') ($total['open'])</a>
-    </li>
-    <li class="$(mode=='closed' and 'selected' or '')">
-      <a href="$changequery(mode='closed', page=None)">$_('Closed') ($total['closed'])</a>
-    </li>
+  <ul class="table-header">
+    <!-- header-left and header-right hacks to make alignment work for now; will change during integration -->
+    <div class="header-left">
+      <li class="$(mode=='open' and 'selected' or '')">
+        <a href="$changequery(mode='open', page=None)">$_('Open') ($total['open'])</a>
+      </li>
+      <li class="$(mode=='closed' and 'selected' or '')">
+        <a href="$changequery(mode='closed', page=None)">$_('Closed') ($total['closed'])</a>
+      </li>
+    </div>
+    <div class="header-right">
+      <li class="mr-dropdown" id="submitter-menu-button">Submitter ▾
+        <div class="mr-dropdown-menu hidden">
+          <header class="dropdown-header">
+              <span>Submitter</span>
+              <button class="dropdown-close">&times;</button>
+          </header>
+          <input type="text" class="filter" placeholder="Filter submitters" id="submitter-filter">
+          <a href="$changequery(submitter=username, page=None)" class="dropdown-item">
+            <span class="$('' if submitter==username else 'visibility-hidden') item-checked">✓</span>
+            <span><b>Submitted by me</b></span>
+          </a>
+        $for s in total['submitters']:
+          $ submitter_name = s.get('submitter', 'default submitter')
+          $ submitter_href = changequery(submitter=submitter_name, page=None)
+          <a href="$submitter_href" class="dropdown-item">
+            <span class="$('' if submitter==submitter_name else 'visibility-hidden') item-checked">✓</span>
+            <span>@$submitter_name</span>
```

Comment:
```suggestion
            <span>$submitter_name</span>
```
These `@` symbols are a little confusing.  Do I need to type `@` when I'm filtering?  Let's just remove them.